scripts/dev/run-test/fm-run-test

```diff
@@ -26,11 +26,11 @@ on_error() {
   cat "$test_out_path"
   echo "## FAIL: $test_name$version_str"
   echo "## LOG FEDIMINTD-0:"
-  cat "$FM_RUN_TEST_TMPDIR/logs/fedimintd-default-0.log"
+  cat "$FM_RUN_TEST_TMPDIR"/devimint-*/logs/fedimintd-default-0.log || true
```

Comment:
devimint should set `FM_LOGS_DIR` as well

Reply:
Yeah, but that's further down the line, so I don't have it here :shrug: .